UnitTests/GitCommands.Tests/Helpers/PathUtilTest.cs

```diff
@@ -104,12 +104,14 @@ public void IsLocalFileTest()
         [TestCase(null, false)]
         [TestCase("", false)]
         [TestCase("    ", false)]
-        [TestCase("http://", true)]
+        [TestCase("http://", false)]
         [TestCase("HTTPS://www", true)]
-        [TestCase("git://", true)]
-        [TestCase("file:", true)]
-        [TestCase("SSH:", true)]
+        [TestCase("git://", false)]
+        [TestCase("file:", false)]
+        [TestCase("SSH:", false)]
         [TestCase("SSH", false)]
+        [TestCase("rtsp://probably.not.a.git.url/default.mp4", false)]
```

Comment:
The method `IsUrl` suggests it will return true for URLs. This actually is a URL, even if not a git one.

Reply:
You're right, I was trying to be too clever. :) I'll remove this test case from this branch and think about how to update it in master (either remove the test case or rename the method)